Modules/Quotas/test/Quotas.Application.Tests/Tests/Identities/GetIdentity/HandlerTests.cs

```diff
@@ -60,12 +59,11 @@ public async void Gets_identity_quotas_by_address()
     public void Fails_when_no_identity_found()
     {
         // Arrange
-        var metricsRepository = A.Fake<IMetricsRepository>();
-        var identitiesRepository = A.Fake<IIdentitiesRepository>();
-        var metricCalculatorFactory = A.Fake<MetricCalculatorFactory>();
-        A.CallTo(() => identitiesRepository.Find(A<string>._, A<CancellationToken>._, A<bool>._)).Returns((Identity)null);
+        var dummyMetricsRepository = A.Dummy<IMetricsRepository>();
+        var stubIdentitiesRepository = A.Fake<IIdentitiesRepository>();
+        A.CallTo(() => stubIdentitiesRepository.Find(A<string>._, A<CancellationToken>._, A<bool>._)).Returns((Identity)null);
 
-        var handler = CreateHandler(identitiesRepository, metricsRepository, metricCalculatorFactory);
+        var handler = CreateHandler(stubIdentitiesRepository, dummyMetricsRepository);
```

Comment:
In general, I would recommend to not pass dummies to the CreateHandler method and instead initialize them in the CreateHandler method instead. Since there is no configuration required, it's no problem and it simplifies the tests even further.

Reply:
The thing is that on line 36 a stub is passed, i.e., there are cases where this is passed. Should I make it optional instead, and create it in the `CreateHandler` method if needed?